test/test-slowparse.js

```diff
@@ -42,6 +42,14 @@ module.exports = function(Slowparse, window, document, validators) {
     });
   });
 
+  test("parsing of valid DOCTYPE with whitespace", function() {
```

Comment:
can we also add the test cases for "comment", "white space, comment" and "comment, white space"? just to cover the spec?